.github/workflows/lint.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
-          node-version: '14.x'
+          node-version: '16.x'
```

Comment:
should we update node-version for all workflows including tests ?

Reply:
@NeOMakinG is on it afaik ^^

Reply:
I guess we should if we want to push package locks using node 16 and npm 8.5